pom.xml

```diff
@@ -89,7 +89,12 @@
 			<version>1.3.RC2</version>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>com.esotericsoftware.yamlbeans</groupId>
+            <artifactId>yamlbeans</artifactId>
```

Comment:
[minor] Is YML a common format in the Java world? Would another format be used more commonly? Is this library standard/good?